dependencies.props

```diff
@@ -1,14 +1,43 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <!-- Package versions used as toolsets --> 
-    <PropertyGroup> 
-        <FeedTasksPackage>Microsoft.DotNet.Build.Tasks.Feed</FeedTasksPackage> 
-        <FeedTasksPackageVersion>1.0.0-prerelease-02121-01</FeedTasksPackageVersion> 
-    </PropertyGroup> 
-    <ItemGroup>
-        <XmlUpdateStep Include="BuildTools">
-            <Path>$(MSBuildThisFileFullPath)</Path>
-            <ElementName>FeedTasksPackageVersion</ElementName>
-            <PackageId>$(FeedTasksPackage)</PackageId>
-        </XmlUpdateStep>
-    </ItemGroup>
+  <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+  <!-- For detailed explanation of the reference versions see:
+
+       https://github.com/dotnet/buildtools/blob/master/Documentation/project-nuget-dependencies.md
+       https://github.com/dotnet/buildtools/blob/master/Documentation/annotated-dependency-props.md
+
+       These ref versions are pulled from https://github.com/dotnet/versions.
+  -->
+  <PropertyGroup>
+    <BuildToolsCurrentRef>c6906dcb2309085e414c5b9691de84fbe2bf5c86</BuildToolsCurrentRef>
+  </PropertyGroup>
+
+  <!-- Package versions used as toolsets --> 
+  <PropertyGroup> 
+    <FeedTasksPackage>Microsoft.DotNet.Build.Tasks.Feed</FeedTasksPackage> 
+    <FeedTasksPackageVersion>1.0.0-prerelease-02121-01</FeedTasksPackageVersion> 
```

Comment:
Hmm, did you run `UpdateDependencies` to try to get this in sync? https://github.com/dotnet/versions/blob/c6906dcb2309085e414c5b9691de84fbe2bf5c86/build-info/dotnet/buildtools/master/Last_Build_Packages.txt shows `1.0.0-prerelease-02124-02`. If `UpdateDependencies` isn't working, then auto-update won't work either.

Reply:
I ran updatedependencies, and it didn't change or complain about anything.  I'll dig into it a bit more

Reply:
User error, I ran the target incorrectly.  fixed now.

Reply:
Thanks for checking, that basically confirms that once the subscription's wired up, it'll work.